<launch>
    <node pkg="dr_controller" type="league_api_interface" name="league_api_interface"/>
</launch>